recipes/mykrobe/build.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-
-$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+wget https://bit.ly/2H9HKTU -O - | tar -vxzf  -
```

Comment:
please use a post-installation hook to download this data after installation and do not include it in this package.

Reply:
Sorry I don't know what you mean by that @bgruening

Reply:
Something like that for example: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/ensembl-vep/post-link.sh

Or a more complicated one: https://github.com/bioconda/bioconda-recipes/blob/master/recipes/r-ngsplot-hg19/post-link.sh

Reply:
May I ask why? It is a panel of variants that program *needs* in order to function correctly. Therefore should it not be included in the package?

Reply:
We want to keep the package size small. All of this needs to be stored and backuped. Data should be distributed separately. You download would add 30MB for every version of your package to the conda channel. Because you have no arch-depending python package, bioconda will create for you 4 packages (py27,35,36,37) - means 4*30MB - just for one singel version.

If you need a permanent storage for your data consider using https://github.com/galaxyproject/cargo-port or Zenodo.

Reply:
Another question about this post-link script. Will the data I am trying to add in the post-link script be accessible to mykrobe at runtime? 

Also I see in the [conda docs](https://conda.io/projects/conda-build/en/latest/source/link-scripts.html) that they strongly discourage the use of these scripts.